brayovsky/yolo
# Ch create wireframe 143118231

#### What does this PR do? 
Creates a wireframe for the project. It contains relevant settings for different contextx that the app will be used, function  declarations for app routes and helper functions, models for the app and authentication.

#### Description of Task to be completed? 
Write all functions for the project without implementation. Use the correct libraries for this.

#### How should this be manually tested? 
Run the server after navigating to the root folder of the project in your terminal.
`python manage.py runserver`
The server should run without any errors albeit the routes do not return anything.

#### Any background context you want to provide? 
None

#### What are the relevant pivotal tracker stories? 
143118231

#### Screenshots (if appropriate) #### Questions:
